tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature

```diff
@@ -185,39 +73,23 @@ Feature: accept/decline shares coming from internal users
     And the user has browsed to the personal sharing settings page
     Then User-based auto accepting checkbox should not be displayed on the personal sharing settings page in the webUI
 
-  @skip @yetToImplement
-  Scenario: Admin disables auto-accept setting again after user enabled personal auto-accept setting
-    Given the setting "Automatically accept new incoming local user shares" in the section "Sharing" has been enabled
-    And user "user1" has logged in using the webUI
-    And the user has browsed to the personal sharing settings page
-    When the user disables automatically accepting new incoming local shares
-    And the user enables automatically accepting new incoming local shares
-    And the administrator disables the setting "Automatically accept new incoming local user shares" in the section "Sharing"
-    And user "user2" shares folder "/simple-folder" with group "grp1" using the sharing API
-    And user "user2" shares file "/testimage.jpg" with user "user1" using the sharing API
-    And the user browses to the files page
-    Then folder "simple-folder (2)" should not be listed on the webUI
-    And file "testimage (2).jpg" should not be listed on the webUI
-    And folder "simple-folder" should be listed in the shared-with-you page on the webUI
-    And file "testimage.jpg" should be listed in the shared-with-you page on the webUI
-    And folder "simple-folder" should be in state "Pending" in the shared-with-you page on the webUI
-    And file "testimage.jpg" should be in state "Pending" in the shared-with-you page on the webUI
-
   Scenario: User receives files when auto accept share is disabled
     Given the setting "shareapi_auto_accept_share" of app "core" has been set to "no"
     And user "user1" has shared file "lorem.txt" with user "user2"
     When the user browses to the shared-with-me page using the webUI
     Then file "lorem.txt" shared by "User One" should be in "Pending" state on the webUI
     When the user browses to the files page
-    Then file "lorem (2).txt" should not be listed on the webUI
+    Then file "lorem.txt" should not be listed on the webUI
+    And folder "Shares" should not be listed on the webUI
 
   Scenario: shared file is in pending state when the Automatically accept incoming shares is disabled
     Given the setting "shareapi_auto_accept_share" of app "core" has been set to "no"
```

Comment:
This is now in the Background section.
There are more scenarios below also like that...